src/solver/batchnorm/backward_ck.cpp

```diff
@@ -201,7 +201,8 @@ bool BnCKBwdBackward::IsApplicable(
         return false;
     if(bn_problem.GetDirection() != miopen::batchnorm::Direction::Backward)
         return false;
-
+    if(!bn_problem.Is2D())
+        return false;
```

Comment:
Should this have always been there?

I would have thought this was needed before these changes as well.

Reply:
yes! current CK batch norm only supports 2D.